recipes/phylics/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 build:
   number: 0
   noarch: python
-  skip: True  # [py < 37]
+  #skip: True  # [py < 37]
```

Comment:
@dpryan79 is it better to remove this line (rather than comment it out)?

Reply:
Yeah, I'll edit the PR real quick